.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "production-eng-service",
   "image": "mcr.microsoft.com/devcontainers/universal:2",
+  // https://github.com/devcontainers/features/issues/580
+  "containerEnv": {
+    "DISABLE_JAVA_BUILD": "true"
+  },
   "features": {
-    "ghcr.io/devcontainers/features/java:1": {
-      "version": "11"
-    },
+    // "ghcr.io/devcontainers/features/java:1": {
+    //   "version": "21"
+    // },
```

Comment:
I think we can remove this comment, since it's an issue that has been already fixed

Reply:
fixed